src/app/lib/layoutGrid.js

```diff
@@ -1,48 +1,69 @@
 import { css } from 'styled-components';
 import {
-  GEL_SPACING,
-  GEL_SPACING_DBL,
-  group2ScreenWidthMax,
+  GEL_MARGIN_BELOW_400PX,
+  GEL_GUTTER_BELOW_600PX,
+  GEL_MARGIN_ABOVE_400PX,
+  GEL_GUTTER_ABOVE_600PX,
+  group2ScreenWidthMin,
   group3ScreenWidthMin,
   group3ScreenWidthMax,
   group4ScreenWidthMin,
   group4ScreenWidthMax,
   group5ScreenWidthMin,
 } from './constants/styles';
 
+const group4ColWidth = `6.75rem`;
```

Comment:
Can these be moved to `constant/styles`?

Reply:
These are very much bespoke to the 1280px/1008px columns rendering logic (due to the margins being exclusive of the grid and the gutters being inclusive. The actual values are `group4ColWidth == 108px` and `group5ColWidth == 110.4` - it is highly unlikely another file will re-use them, I'd even want to question why they would need to be used in the future so would such keeping them here.